.github/workflows/build-and-test-multiplatform.yml

```
@@ -0,0 +1,350 @@
+          mkdir build
+          cd build
+          cmake -T host=x64 -D USE_FREETYPE=OFF -D USE_TK=OFF -D BUILD_USE_PCH=ON -D BUILD_OPT_PROFILE=Production -D BUILD_INCLUDE_SYMLINK=ON -D CMAKE_BUILD_TYPE=Release -D 3RDPARTY_DIR=${{ github.workspace }}/3rdparty-vc14-64 ..
+  
+    - name: Build OCCT
+      run: |
+          cd build
+          cmake --build . --config Release -- /m
+
+  prepare-and-build-macos-x64:
+    name: Prepare and Build on macOS with Clang (x64)
+    runs-on: macos-15
```

actionlint

label "macos-15" is unknown. available labels are "windows-latest", "windows-latest-8-cores", "windows-2022", "windows-2019", "ubuntu-latest", "ubuntu-latest-4-cores", "ubuntu-latest-8-cores", "ubuntu-latest-16-cores", "ubuntu-24.04", "ubuntu-22.04", "ubuntu-20.04", "macos-latest", "macos-latest-xl", "macos-latest-xlarge", "macos-latest-large", "macos-14-xl", "macos-14-xlarge", "macos-14-large", "macos-14", "macos-14.0", "macos-13-xl", "macos-13-xlarge", "macos-13-large", "macos-13", "macos-13.0", "macos-12-xl", "macos-12-xlarge", "macos-12-large", "macos-12", "macos-12.0", "macos-11", "macos-11.0", "self-hosted", "x64", "arm", "arm64", "linux", "macos", "windows". if it is a custom label for self-hosted runner, set list of labels in actionlint.yaml config file

(runner-label)